Sources/Contour/IBusFactory.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Contour
+{
+    using System;
+    using Configuration;
+
+    /// <summary>
+    /// The BusFactory interface.
+    /// </summary>
+    public interface IBusFactory
```

Comment:
IBusFactory has been introduced to describe the common behavior of Bus objects' construction. It is needed to support testing as well.